test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1308,6 +1309,38 @@ def revert_moms(self, force=False):
         for mom in self.moms.values():
             self.revert_mom(mom, force)
 
+    def add_mgrs_opers(self, server):
+        """
+        Adding manager and operator users
+        """
+        if not self.use_cur_setup:
+            try:
+                # Unset managers list
+                server.manager(MGR_CMD_UNSET, SERVER, 'managers', sudo=True)
+                # Unset operators list
+                server.manager(MGR_CMD_UNSET, SERVER, 'operators', sudo=True)
+            except PbsManagerError as e:
+                self.logger.error(e.msg)
+        attr = {}
+        current_user = pwd.getpwuid(os.getuid())[0] + '@*'
+        mgrs_opers = {"managers": [current_user, str(MGR_USER) + '@*'],
+                      "operators": [str(OPER_USER) + '@*']}
+        server_stat = server.status(SERVER, ["managers", "operators"])
+        if len(server_stat) > 0:
+            server_stat = server_stat[0]
+        for role, users in mgrs_opers.items():
+            if role not in server_stat:
+                attr[role] = (INCR, ','.join(users))
+            else:
+                add_users = []
+                for user in users:
+                    if user not in server_stat[role]:
+                        add_users.append(user)
+                if len(add_users) > 0:
+                    attr[role] = (INCR, ",".join(add_users))
+        if len(attr) > 0:
+            self.server.manager(MGR_CMD_SET, SERVER, attr, sudo=True)
```

Comment:
you are passing in 'server' object so you don't need to call it as "self.server". If you take my suggestion for converting this into a class method, then you'll have to use 'cls.server' everywhere in the method